administrator-guides/custom-fields/README.md

```diff
@@ -57,4 +63,6 @@ In the example above we set two new fields with the following properties:
 
 - **private**: defines the field as private, so only users with `view-full-other-user-info` permission can see this field when viewing this user's profile.
 
+- **sendToIntegrations**: define the field as shareable with external applications, such as Omnichannel integrations.
```

Comment:
At some point I think we'll need to describe better which integrations the field will be sent to.. but for now I think Omnichannel integrations is fine 👍